modules/core/02-client/types/proposal.go

```diff
@@ -5,7 +5,7 @@ import (
 
 	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
-	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
```

Comment:
this is a teeny bit hacky:

The proper way could be to fully refactor all use of governance, but it is my belief that this works.